packages/mobile_app/lib/screens/registration/screen.dart

```diff
@@ -0,0 +1,226 @@
+              child: Text(
+                "offline",
+                style: TextStyle(color: AppColors.textDisabled),
+              ),
+            ),
+            UserInfo(
+              name: UserInfoField(
+                  textObject: name,
+                  maxLength: 100,
+                  hint: lang.name,
+                  validator: (String text) {
+                    int min = 2;
+                    if (text.isEmpty || text.length < min)
+                      return "${lang.atLeastXsymbolsNeeded} $min";
+                    return null;
+                  }),
+              month: UserInfoField(
+                  textObject: month,
+                  maxLength: 2,
+                  keyboardType: TextInputType.datetime,
+                  hint: "mm",
+                  validator: (String text) {
+                    int min = 2;
+                    if (text.isEmpty || text.length < min)
+                      return "${text.length}/$min";
+                    if (int.parse(text) > 12 || int.parse(text) < 1) return "x";
+                    return null;
+                  }),
+              day: UserInfoField(
+                  textObject: day,
+                  maxLength: 2,
+                  keyboardType: TextInputType.datetime,
+                  hint: "dd",
+                  validator: (String text) {
+                    int min = 2;
+                    if (text.isEmpty || text.length < min)
+                      return "${text.length}/$min";
+                    if (int.parse(text) > 31 || int.parse(text) < 1) return "x";
+                    return null;
+                  }),
+              year: UserInfoField(
+                  textObject: year,
+                  maxLength: 4,
+                  keyboardType: TextInputType.datetime,
+                  hint: "yyyy",
+                  validator: (String text) {
+                    int min = 4;
+                    if (text.isEmpty || text.length < min)
+                      return "${text.length}/$min";
+                    return null;
+                  }),
+              sex: UserInfoPicker(
+                items: indexToSex,
+                indexObject: sex,
+              ),
+              country: UserInfoField(
+                  textObject: country,
+                  maxLength: 50,
+                  hint: lang.birthcountry,
+                  validator: (String text) {
+                    int min = 2;
+                    if (text.isEmpty || text.length < min)
+                      return "${lang.atLeastXsymbolsNeeded} $min";
+                    return null;
+                  }),
+              place: UserInfoField(
+                  textObject: place,
+                  maxLength: 100,
+                  hint: lang.birthplace,
+                  validator: (String text) {
+                    int min = 2;
+                    if (text.isEmpty || text.length < min)
+                      return "${lang.atLeastXsymbolsNeeded} $min";
```

Comment:
This field is also not mandatory

Reply:
You mean, user can enter empty country and place?

Reply:
@bgoncharuck Exactly